easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-0.9.0-gompi-2019b.eb

```diff
@@ -0,0 +1,87 @@
+easyblock = 'CMakeMakeCp'
+
+name = 'RAxML-NG'
+version = '0.9.0'
+
+local_terrap_tree = '6dcb061ceb6ed41a7ebe1a8af1f3354e0126125c'
+local_pllmod_tree = '840a19525ed133ad243afb23124911b849d4685d'
+local_libpll_tree = '6874327c9ca9544e27b426c0e7205c8b6f4ed297'
+
+homepage = "https://github.com/amkozlov/raxml-ng"
+description = """RAxML-NG is a phylogenetic tree inference tool which
+uses maximum-likelihood (ML) optimality criterion. Its search heuristic
+ is based on iteratively performing a series of Subtree Pruning and Regrafting
+ (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/amkozlov/raxml-ng/archive/'],
+        'filename': '%(version)s.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/amkozlov/terraphast-one/archive/'],
+        'download_filename': local_terrap_tree + '.zip',
+        'filename': 'terraphist-one.zip',
+    },
+    {
+        'source_urls': ['https://github.com/ddarriba/pll-modules/archive/'],
+        'download_filename': local_pllmod_tree + '.zip',
+        'filename': 'pll-modules.zip',
+    },
+    {
+        'source_urls': ['https://github.com/xflouris/libpll-2/archive/'],
+        'download_filename': local_libpll_tree + '.zip',
+        'filename': 'libpll-2.zip',
+    },
+]
+
+# copy dependencies into place
+preconfigopts = ("cp -a %(builddir)s/terraphast-one-" + local_terrap_tree +
+                 "/* %(builddir)s/raxml-ng-%(version)s/libs/terraphast/ && ")
+
+preconfigopts += ("cp -a %(builddir)s/pll-modules-" + local_pllmod_tree +
+                  "/* %(builddir)s/raxml-ng-%(version)s/libs/pll-modules/" +
+                  " && ")
+
+preconfigopts += ("cp -a %(builddir)s/libpll-2-" + local_libpll_tree +
+                  "/* %(builddir)s/raxml-ng-%(version)s/libs/pll-modules/" +
+                  "libs/libpll/ && ")
+
+checksums = [
+    '43b95ee1bddae0daee84644e9ee760a77f28bb16e2071e95cca79a30f39373e5',
+    'eef5e2f365619da3bb1679f07920b29dfcd5202191fdf37b9f8fec4a52c1286b',
+    '486f55cc2b4a63ddba8d00db79afcbf03f2499e35687eef56e4730f23d96b90a',
+    '6c90af957d4b06b88ef6568f6ba53ac3507285d042dc9ebc95f9837a7077d044',
+]
+
+build_type = 'Release'
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('CMake', '3.15.3'),
+    ('gtest', '1.8.1', '', ('GCCcore', '8.3.0')),
```

Comment:
why not simply `('gtest', '1.8.1')`?

Reply:
With ('gtest', '1.8.1') and with no extra parameter it tries to find _gtest-1.8.1-gompi_  which doesn't exist and fails.

That's why I force the use of _gtest-1.8.1-Gcccore-8.3.0_, which is part of this pr and replicated from the _gtest-1.8.1-Gcccore-8.2.0_.

Reply:
that shouldn't be necessary, `GCCcore` is a subtoolchain of `gompi`... what version of EB are you using?

Also, I just noticed that https://github.com/easybuilders/easybuild-easyconfigs/pull/10216 adds gtest 1.10.0 for the same toolchain, please use the same as we enforce a single dependency variant per toolchain generation 

(I'm sorry I missed that before)

Reply:
Hello Miguel,

Just adding more information about the version and the error I get without specifying the toolchain for gtest.

This is EasyBuild 4.1.0 (framework: 4.1.0, easyblocks: 4.1.0) on host b10d98675de4.

eb -r RAxML-NG-0.9.0-gompi-2019b.eb gtest-1.8.1-GCCcore-8.3.0.eb
== temporary log file in case of crash /tmp/eb-C8Kb_o/easybuild-nIQeai.log
== resolving dependencies ...
ERROR: Missing dependencies: gtest/1.8.1-gompi-2019b (no easyconfig file or existing module found)

Reply:
what about if you do, without specifying the toolchain for `gtest`, `eb RAxML-NG-0.9.0-gompi-2019b.eb -r .`? (so that the `gtest` easyconfig is in the robot path)

Reply:
also, now both versions of `gtest` are in the PR, you can remove the old one with something like

 `eb --update-pr :gtest-1.8.1-GCCcore-8.3.0.eb --pr-commit-msg="remove old version of gtest"`

(note the colon before the filename, which means delete that file)

Reply:
That worked! So I removed the toolchain from the dependency and I removed the gtest-1.8.1 easyconfig from the pr too.